src/applications/facility-locator/utils/formatServiceName.js

```diff
@@ -12,12 +14,24 @@ export const formatServiceName = service => {
     .toLowerCase()
     .replace(/veteran/g, 'Veteran')
     .replace(/covid19 vaccine/g, 'COVID-19 vaccines')
-    .replace(/E benefits/g, 'eBenefits')
-    .replace(/V a home loan help/g, 'VA home loan help')
+    .replace(/e benefits/g, 'eBenefits')
     .replace(
       /Integrated disability evaluation system assistance/g,
       'Integrated Disability Evaluation System assistance (IDES)',
     )
     .trim();
+  if (lowerCaseService === 'eBenefits registration assistance') {
```

Comment:
This seems like a good candidate for a switch statement instead of if / else if

Reply:
just finished converting over to a switch statement.